packages/mobile/src/identity/verification.ts

```diff
@@ -83,14 +84,19 @@ export interface AttestationCode {
 
 export function* fetchVerificationState() {
   try {
-    const account: string = yield call(getConnectedUnlockedAccount)
+    const account: string = yield call(getConnectedAccount)
     const e164Number: string = yield select(e164NumberSelector)
     const contractKit = yield call(getContractKit)
     const attestationsWrapper: AttestationsWrapper = yield call([
       contractKit.contracts,
       contractKit.contracts.getAttestations,
     ])
 
+    // Lock and ask user for a PIN because we
+    // want to reset timeout for unlocked account
+    clearPasswordCaches()
+    yield call(unlockAccount, account, true)
+
```

Comment:
This causes the user to be asked for their PIN when starting verification, right?

Reply:
yeap 🙂